site-src/index.md

```diff
@@ -94,8 +94,32 @@ This flexibility allows the API to adapt to vastly different
 organizational models and implementations while remaining a portable and
 standard API.
 
+## What's the difference between Gateway API and an API Gateway?
 
-## Who is working on Gateway?
+The concept of a Gateway is a networking component that allows data to transfer from one
+network to another network.
+
+An API Gateway is a general concept that describes anything that exposes capabilities of a backend service, while providing extra capabilities for traffic routing and manipulation, such as load balancing, request and response transformation, and sometimes more advanced features like authentication and authorization, rate limiting, and circuit breaking.
+
+Most Gateway API implementations are API Gateways to some extent, but not all API Gateways are Gateway API implementations.
```

Comment:
I think this particular sentence may work best at either the beginning or end of the section. That would mean we'd end up with one of the following orders:

1. An API Gateway is...
2. Gateway API is...
3. Most Gateway API implementations are API Gateways to some extent, but not all API Gateways are Gateway API implementations.

OR

1. Most Gateway API implementations are API Gateways to some extent, but not all API Gateways are Gateway API implementations.
2. An API Gateway is...
3. Gateway API is...

Reply:
Let's try the first